src/IssueViz.Security/Resources.resx

```diff
@@ -117,6 +117,9 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="ERR_CannotNavigate" xml:space="preserve">
+    <value>Cannot navigate to step. The source code is different from the analyzed version</value>
```

Comment:
I think `Location` would be better than `step`, and since there is more than one sentence the second sentence should have a full stop tool.
Also, it helps to prefix the resource name with the component that uses it, particularly since this is in the top-level resource file.